src/WebJobs.Script/Rpc/WebHostLanguageWorkerChannelManager.cs

```diff
@@ -139,8 +148,20 @@ internal void ScheduleShutdownStandbyChannels()
             }
         }
 
+        public void ShutdownWorkers(WorkerShutdownEvent restartEvent)
```

Comment:
This shouldn't be an event handler. It should be a method to restart the workers.